docs/Users_Guide/tc-pairs.rst

```diff
@@ -258,52 +285,75 @@ ____________________
 
 .. code-block:: none
 
- diag_name = [];
-
-The **diag_name** entry specifies a comma-separated list of strings for the tropical cyclone diagnostics of interest. This applies when the **-tcdiag** and/or **-lsdiag** command line options have been used to provide storm diagnostics data. If a non-zero list of diagnostic names is specified, only those diagnostics appearing in the list are written to the TCDIAG output line type. If defined as an empty list (default), all diagnostics found in the input are written to the TCDIAG output lines.
-
-A TCMPR line is written to the output for each track point. If diagnostics data is also defined for that track point, a TCDIAG line is written immediately after the corresponding TCMPR line. The contents of that TCDIAG line is determined by diagnostic names requested in the **diag_name** entry.
+ diag_info_map = [
+    {
+       diag_source    = "CIRA_DIAG_RT";
+       track_source   = "GFS";
+       field_source   = "GFS_0p50";
+       match_to_track = [ "GFS" ];
+       diag_name      = [];
+    },
+    {
+       diag_source    = "SHIPS_DIAG_RT";
+       track_source   = "OFCL";
+       field_source   = "GFS_0p50";
+       match_to_track = [ "OFCL" ];
+       diag_name      = [];
+    }
+ ];
+
+A TCMPR line is written to the output for each track point. If diagnostics data is also defined for that track point, a TCDIAG line is written immediately after the corresponding TCMPR line. The contents of that TCDIAG line is determined by the **diag_info_map** entry.
+
+The **diag_info_map** entries define how the diagnostics read with the **-diag** command line option should be used. Each array element is a dictionary consisting of entries for **diag_source**, **track_source**, **field_source**, **match_to_track**, and **diag_name**.
+
+The **diag_source** entry is one of the supported diagnostics data sources. The **track_source** entry is a string defining the ATCF ID of the track data used to define the locations at which diagnostics are computed. This string is written to the **TRACK_SOURCE** column of the TCDIAG output line. The **field_source** entry is a string describing the gridded data over which the diagnostic are computed. This string is written to the **FIELD_SOURCE** column of the TCDIAG output line type. The **match_to_track** entry specifies a comma-separated list of strings defining the ATCF ID(s) of the tracks to which these diagnostic values should be matched. The **diag_name** entry specifies a comma-separated list of strings for the tropical cyclone diagnostics of interest. If a non-zero list of diagnostic names is specified, only those diagnostics appearing in the list are written to the TCDIAG output line type. If defined as an empty list (default), all diagnostics found in the input are written to the TCDIAG output lines.
```

Comment:
Minor edit for clarity/accuracy:

Suggest changing: 
The **field_source** entry is a string describing the gridded data over which the diagnostic are computed.

to:
The **field_source** entry is a string describing the gridded data from which the diagnostics are computed.

Reply:
Done. Fixed by [this commit](https://github.com/dtcenter/MET/commit/d7c04bfe7eba02d729c6feeb1ac39411a62c4d37). For one line edits, please consider "proposing" the exact change you'd like to see so I can just click the button to accept the change.